### PR TITLE
mingw: compile the mingw-w64 companion tools as well

### PR DESCRIPTION
**Pull request by lihanglin**
There are some useful tools (widl, gendef ...) provided by mingw-w64
and do not waste the developers' works.

### CODE REVIEW

File: scripts/build/libc/mingw.sh [hunk 1]
```diff
@@ -83,6 +83,33 @@ do_check_mingw_vendor_tuple()
...
+
+	CT_mkdir_pushd "${CT_BUILD_DIR}/build-mingw-w64-tools/${f}"
+
+	CT_DoExecLog CFG        \
+	    "${CT_SRC_DIR}/mingw-w64-${CT_WINAPI_VERSION_DOWNLOADED}/mingw-w64-tools/${f}/configure" \
+            --build=${CT_BUILD} \
+            --host=${CT_HOST} \
+            --target=${CT_TARGET} \
+            --program-prefix=${CT_TARGET}- \
+	    --prefix="${CT_PREFIX_DIR}"
```
Comment (stilor) [inline scripts/build/libc/mingw.sh]: Indentation (tab vs space)

File: scripts/build/libc/mingw.sh [hunk 2]
```diff
@@ -109,6 +136,8 @@ do_libc() {
     CT_DoLog EXTRA "Installing mingw-w64-crt"
     CT_DoExecLog ALL ${make} install DESTDIR=${CT_SYSROOT_DIR}
 
+    do_mingw_tools
```
Comment (stilor) [inline scripts/build/libc/mingw.sh]: I think this should be an option in config/libc/mingw.in.

**Comment by lihanglin** [comment]
Hi Neyman,

Thanks for reviewing.
I revised this patch with your recommendation and help me review it again please.

**Comment by stilor** [comment]
Thanks! Just add a signed-off-by line to the commit message, and it's good to go.

**Comment by lihanglin** [comment]
Hi Neyman,

Yes.
complete with sign-off.
Thank you.

**Pull request merged**